### DISCUSSION TITLE
problemas com os arquivos index, products

### DISCUSSION THREAD

**Discussion by MarcioFreitasUFSC21** [balta-io/forum#306] [Tira Dúvidas] [answered]
boa tarde, professor
professor, fiz a inclusão do arquivo index.js e compiei os códigos do arquivo app.js, bem como o arquivo products.js,mas minha API não rodou mais; no vsCode, há como se o houvesse erro do ";" após o "use strict". segui todos os passos que o senhor fez, mas há esse problema. Como posos resolver esse problema?
Desde já, agradeço-lhe  a atenção e fico no aguardo.
Att.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @MarcioFreitasUFSC21, como vai?

Consegue mandar um print do código com erro?

**Reply by MarcioFreitasUFSC21** [reply]
claro, professor!
estão em anexo alguns prints.
 o vsCode alerta de que o comando exports.post( ...) não é uma função.
![image](https://user-images.githubusercontent.com/79476199/231805600-e777c24a-fc44-4850-b328-48d5cd884987.png)
![image](https://user-images.githubusercontent.com/79476199/231805841-ae214f5a-2712-42fa-9fac-d8750e87eb2a.png)
![image](https://user-images.githubusercontent.com/79476199/231805918-ea404ebc-634e-4aa1-95e9-a60f7af6250c.png)

**Comment by MarcioFreitasUFSC21** [comment]
pronto, professor!consegui resolver o problema!estava cansado e não havia visto onde estava o erro.Valeu!

**Comment by MarcioFreitasUFSC21** [comment]
boa noite, professor! professor, estou novamente com alguns problemas e nao sei ocmo resolvê-los!
na parte de cadastro de produtos. aqui em casa, minha aplicação está rodando, mas quando tento testar pelo postman, não dá certo.Poder-me-ia ajudar a resolver esse problema?
![image](https://user-images.githubusercontent.com/79476199/233806689-46af87f7-90e9-426e-90ee-63e0cc470ad2.png)
![image](https://user-images.githubusercontent.com/79476199/233806728-7c84d8fc-bb29-46f7-9d75-8137a5c97c39.png)
![image](https://user-images.githubusercontent.com/79476199/233806749-384e66bf-4d92-4689-8c70-af1a7304750c.png)

**Reply by andrebaltieri** [reply]
Bom dia, @MarcioFreitasUFSC21, como vai?

Claro, qual problema está tendo? Consegue dizer em qual linha do seu código ele ocorre e quais mensagens ele te retorna?

**Reply by andrebaltieri** [reply]
Se puder fazer um passo-a-passo de como chegar até o erro, descrever onde (qual tela/situação) ele ocorre...

**Reply by MarcioFreitasUFSC21** [reply]
certo, é o seguinte: consegui resolver o erro anterior, pois havia a mudado a ordem dos itens do produto, mas, ao deixar como o senhor havia feito no exemplo, retornou uma falha de cadastro de produto.como seguem as fotos.
inicialmente após as suas orientações, deixei minha api rodando na porta 3000 e server online
![image](https://user-images.githubusercontent.com/79476199/234101307-9564e911-a669-4a7a-8284-25dd22c4ce1e.png)
em seguida, abri o postman e tentei fazer o devido preenchimento dos itens do produto:
![image](https://user-images.githubusercontent.com/79476199/234101538-d0845320-6930-4568-8562-a3b10276588d.png)
em seguida, lancei uma requisição pelo postman para retornar a mensagem de cadastro de produto, mas houve o retorno da mensagem falha no cadastro de produto e ao lado o erro.
![image](https://user-images.githubusercontent.com/79476199/234101882-d72d1b25-a89b-4a22-b81b-fb11e24b6c0a.png)
com isso, professor, há somente o retorno da mensagem de falha de cadastro de produto
![image](https://user-images.githubusercontent.com/79476199/234102157-ca2e6cab-a63a-41d3-a459-f81e43b91998.png)
o que o senhor sugere? estou analisando o porquê de o programa não entrar na mensagem de sucesso de cadastro. Qualquer coisa, fico no aguardo.

**Reply by MarcioFreitasUFSC21** [reply]
vou neviar via email para o senhor os passos com as devidas figuras em tamanho maior, tudo bem?

**Reply by andrebaltieri** [reply]
@MarcioFreitasUFSC21 pode enviar aqui mesmo, só colar as imagens... Descreve passo a passo com detalhes do que precisa... Assim pode ajudar outras pessoas também.
